installer/x86_64/install.sh

```diff
@@ -206,6 +289,55 @@ create_demo_gpt_partition()
         done < $tmpfifo
     fi
 
+    # Clean up block device that contains ONIE
+    local spec_blk_dev=$(echo $blk_dev | sed -e 's#/dev/##')
+
+    [ -b "/dev/$spec_blk_dev" ] || {
+        echo "Error: Unable to determine block device of ONIE install"
+        exit 1
+    }
+
+    if [ "$firmware" = "bios" ] ; then
+       # When a NOS partition is deleted, the grub entry in MBR will no longer
```

Comment:
Indent inside this `if` statement is only 3 spaces. Please adjust to 4 spaces for consistency.